app/views/publications/v2/monthly_statistics/show.html.erb

```diff
@@ -0,0 +1,247 @@
+
+    <p class="govuk-body">This table counts candidates by their course phase. ‘Course phase’ refers to whether a course trains candidates to teach in primary or secondary education. Further education applications are not included in these statistics.</p>
+    <p class="govuk-body">Candidates may make different applications to more than one course phase. These candidates are counted under all course phases that apply. This means that the total of these rows does not equal the total in the ‘Candidate headline statistics’ section.</p>
+  </div>
+</div>
+
+<%= render Publications::DataTableComponent.new(caption: "Table 6.1: #{@presenter.by_phase[:title]}", title: t('publications.itt_monthly_report_generator.phase.subtitle'), data: @presenter.by_phase[:data]) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l govuk-!-padding-top-4" id="downloads">8. Download the data</h2>
+
+    <p class="govuk-body">You can download all of the data on this page in CSV (comma separated values) format, which can be opened in a spreadsheet and used in other analysis tools.</p>
+
+    <% @csv_export_types_and_sizes.each do |export_type, size| %>
+      <p class='govuk-body'>
+      <%= govuk_link_to t("#{export_type}.label") + " #{size.to_fs(:human_size)}", publications_monthly_report_download_path(export_type: export_type, month: @presenter.month, format: :csv) %>
+       </p>
+    <% end %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l govuk-!-padding-top-4" id="footnotes">9. Footnotes</h2>
+
+    <p class="govuk-body">The Apply for teacher training service is the sole service for postgraduate Initial Teacher Training (ITT) applications in England to Higher Education Institution (HEI), School Centred Initial Teacher Training (SCITT)-based, School Direct, School Direct (salaried) and Teaching Apprenticeship courses.</p>
+
+    <p class="govuk-body">
+      Deferred candidates are counted in a separate column in the tables.
+    </p>
+
+    <% if @presenter.deferred_applications_count.zero? %>
+      <p class="govuk-body">
+        They’re counted in the statistics report for the academic year in which they start their training after their offer is re-confirmed, as well as the year in which they apply.
+      </p>
+
+      <p class="govuk-body">
+        This means that anyone who applied this year but deferring until next year will be counted in next year’s report, as well as this one.
+      </p>
+    <% end %>
+
+    <p class="govuk-body">
+      Deferred candidates are counted in the academic year in which they start their course after training providers have re-confirmed their deferred offer. Providers need to do this before candidates intend to start their training.
+    </p>
+
+    <p class="govuk-body">
+      Statistical releases from previous months during ITT2022 can be found here:
```

Comment:
@MylesJarvis Where do we now want to send users – to gov or our previous years?

Reply:
I think we should default to our service, given the incomplete nature of last cycle's data on gov uk at the minute? We'd need to amend this copy to say something like:
"The ITT2022 statistical release can be found here:"